fe/fe-core/src/main/java/com/starrocks/analysis/ShowRoutineLoadTaskStmt.java

```diff
@@ -150,7 +150,7 @@ public ShowResultSetMetaData getMetaData() {
 
     @Override
     public RedirectStatus getRedirectStatus() {
```

Comment:
How to handle the query when the follower is lagging from the master?
If you are all forward the request to the master, we should not make the user use the follower.

Reply:
SHOW LOAD/SHOW ROUTINE LOAD is load related meta. The current process of these two stmts is forward to master. This change is to make sure that the connected follower has replicated the latest data from master before returning to user, so that the newly loaded data can be queried right after the load state is finished.